homeassistant/components/conversation/__init__.py

```diff
@@ -337,10 +345,21 @@ async def websocket_hass_agent_debug(
                         }
                         for entity_key, entity in result.entities.items()
                     },
+                    # Entities/areas/etc. that would be targeted
                     "targets": {
                         state.entity_id: {"matched": is_matched}
                         for state, is_matched in _get_debug_targets(hass, result)
-                    },
+                    }
+                    if (not result.unmatched_entities)
```

Comment:
Let’s not do inline if statements. Hard to read

Reply:
Refactored